cmake/modules/FindPTex.cmake

```diff
@@ -27,6 +27,8 @@ if (WIN32)
             "$ENV{PROGRAMFILES}/Ptex/include"
             /usr/include
             DOC "The directory where Ptexture.h resides")
+    # TODO: figure out a way to add windows support for PTEX_PREFER_STATIC
+    # (since both static and dynamic libs can have a ".lib" suffix)
```

Comment:
Maybe ask the Ptex devs to name the libs according to some convention, like boost's?